README.rst

```diff
@@ -40,14 +40,57 @@ USAGE
 
 If you want to learn more about coala-bears, its functionality and its usage,
 please take a look at our
-`tutorial <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#what-is-a-bear>`__.
+`tutorial <http://coala.readthedocs.org/en/latest/index.html>`__.
```

Comment:
The link was to the how to write bears section, and if we say that this is tutorial, it should link to the tutorial.